METIS/METIS_DET_IMG_LM.yaml

```diff
@@ -10,10 +10,8 @@ properties:
   temperature: -230
   dit: "!OBS.dit"
   ndit: "!OBS.ndit"
-  width: 256
```

Comment:
do these numbers correspond to the dimensions of a convolution kernel? If they refer to the detector, they should be 2048, no?

Reply:
Red means it was removed in the commit, so `width` and the rest do not exist any more. They were related to the `DetectorWIndow` effect, which enables simulating just a fraction of the detector. I removed that effect as the detector is small anyway (it is useful for MICADO, for instance). It can always be reinstated.